### PR TITLE
Authentication Done

### PR DESCRIPTION
**Pull request by Aarakka12**

### CODE REVIEW

File: app/models/current.rb
Comment (mgandhi-gammastack) [inline app/models/current.rb:1]: What is the need of this?

Comment (Aarakka12) [reply app/models/current.rb:1]: Sir, I have made this current model to store the current user

File: app/mailers/user_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class UserMailer < ApplicationMailer
+
+  # Subject can be set in your I18n file at config/locales/en.yml
+  # with the following lookup:
+  #
+  #   en.user_mailer.confirmation.subject
+  #
+  default from: User::MAILER_FROM_EMAIL
```
Comment (mgandhi-gammastack) [inline app/mailers/user_mailer.rb:8]: we do not need to use a constant here, and not from user model. if you want to create a constant create here or in application mailer

Comment (Aarakka12) [reply app/mailers/user_mailer.rb:8]: okay sir

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
 <html>
   <head>
-    <title>FoodDeliveryApplication</title>
+    <title>FoodDeliveryApp</title>
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
```
Comment (mgandhi-gammastack) [inline app/views/layouts/application.html.erb:10]: Please use bootstrap gem for this instead of cdn

**Review by mgandhi-gammastack — COMMENTED** [review]
THIS WHOLE CODE LOOKS COPY PASTED AND I DONT THINK YOU EVEN UNDERSTAND HALF OF WHAT CODE IS ACTUALLY DOING.

you should not follow any article mindlessly, you should also consider the complexity and how much you will able to grasp from that.

File: .gitignore [hunk 3]
```diff
@@ -34,3 +34,5 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+# .env
```
Comment (mgandhi-gammastack) [inline .gitignore:37]:
```suggestion
.env
```

**Review by mgandhi-gammastack — APPROVED** [review]

**Pull request merged**